### ISSUE TITLE
Add ARIA role comment to alert code snippet

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/component-library#189]
https://github.com/sul-dlss/component-library/blob/644eb2ca06a61d3efe38e123bef7caac2c63da2e/alerts/index.html#L107C9-L107C20
This alert style is often used within an application body to visually distinguish content to the user. Including the `role="alert"` in these contexts is inappropriate, since this role is only for time-sensitive information that the screen reader should announce immediately. See [ARIA: alert role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/alert_role). To avoid confusion, we should add a comment to the alert code snippet stating that the `role="alert"` is optional, depending on its context.
